addons/website/models/website_page.py

```diff
@@ -262,6 +263,18 @@ def get_valid_page_url(self, page_url, website_id=False):
         url = '/' + slugify(page_url, max_length=1024, path=True)
         return self.env['website'].with_context(website_id=website_id).get_unique_path(url)
 
+    def action_manage_website_pages(self):
+        website = self.env['website'].get_current_website()
+        return {
+            'name': _('Website Pages'),
+            'type': 'ir.actions.act_window',
+            'res_model': 'website.page',
+            'view_mode': 'tree',
+            'view_id': self.env.ref('website.website_pages_tree_view').id,
+            'domain': website.website_domain(),
```

Comment:
probably not website_domain but just domain with website_id (not or false)
Otherwise we have duplicated (generic and specific page for the same URL) which doesn't really makes sens anymore now. Before it had sense because:
1. We had a specific display for this case, I'll let you see on master -> the generic was grayed out if there was a specific for that URL and not clickable
2. The sort by name removed the generic
Since we lost all those "features", we should probably never show generic + specific unless we sleect "all websites"
![image](https://user-images.githubusercontent.com/30048408/174284252-8e84753e-808c-442d-8246-bc0926b199cb.png)

Reply:
OK I updated the code this way (with only website_id) "FOR NOW" and I will check if there is a way to have something as in master (at least to reference the generic page) :+1:

Reply:
Yep, ideally it would be great to filter the results to have generic + specific with a filter_duplicate, so if we have

A
B
B'
C
D
D'

The page manager for website 1 shows
A
B'
C
D'

Right now it will only show B' and D', but the user can select "All websites" to see all the pages  I guess :+1:

Reply:
I added a point in the PAD in the important section about this for post merge